ti/instrumentation/utils.go

```diff
@@ -451,6 +454,18 @@ func filterTestsAfterSelection(selection ti.SelectTestsResp, testGlobs string) t
 	return selection
 }
 
+func isExcluded(class string) bool {
+	if os.Getenv("TI_SKIP_EXCLUDE_VENDOR") == "true" {
```

Comment:
read the env map of the step to get the env variables

Reply:
done